extensions/amp-story-interactive/0.1/amp-story-interactive-results-detailed.js

```diff
@@ -36,7 +57,74 @@ export class AmpStoryInteractiveResultsDetailed extends AmpStoryInteractive {
 
   /** @override */
   buildComponent() {
-    this.rootEl_ = htmlFor(this.element)`<p>Detailed results component</p>`;
+    this.rootEl_ = buildResultsDetailedTemplate(this.element);
+    this.buildTop();
     return this.rootEl_;
   }
+
+  /** @override */
+  onInteractiveReactStateUpdate(interactiveState) {
+    const components = Object.values(interactiveState);
+    if (this.selectedResultEls_) {
+      // Function not passed in directly to ensure "this" works correctly
+      components.forEach((e) => this.updateSelectedResult_(e));
+    } else {
+      this.initializeSelectedResultContainers_(components);
+    }
+
+    super.onInteractiveReactStateUpdate(interactiveState);
+  }
+
+  /**
+   * Create and store elements that will show results
+   * for each interactive component.
+   *
+   * @param {!Array<Object>} components
+   * @private
+   */
+  initializeSelectedResultContainers_(components) {
+    this.selectedResultEls_ = {};
+    const detailedResultsContainer = this.rootEl_.querySelector(
+      '.i-amphtml-story-interactive-results-detailed'
+    );
+    components.forEach((e) => {
+      const container = document.createElement('div');
+      container.classList.add(
+        'i-amphtml-story-interactive-results-selected-result'
+      );
+      setImportantStyles(container, {
+        'height': '5em',
+        'width': '5em',
+      });
```

Comment:
Is there a reason why you don't have this on the CSS?

Reply:
I have it there since in a future PR the height and width will be set randomly so I made it 5em for now as a placeholder